wiz-admission-controller/templates/cronjobmanager.yaml

```diff
@@ -0,0 +1,205 @@
+              {{- with (coalesce .Values.global.clusterTags .Values.webhook.clusterTags) }}
+                - --cluster-tags
+                - {{ . | toJson | quote }}
+              {{- end }}
+              {{- with (coalesce .Values.global.subscriptionTags .Values.webhook.subscriptionTags) }}
+                - --subscription-tags
+                - {{ . | toJson | quote }}
+              {{- end }}
+              env:
+              {{- if not .Values.wizApiToken.usePodCustomEnvironmentVariablesFile }}
+                - name: WIZ_CLIENT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.secretApiTokenName" . | trim }}
+                      key: clientId
+                      optional: false
+                - name: WIZ_CLIENT_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.secretApiTokenName" . | trim }}
+                      key: clientToken
+                      optional: false
+              {{- end }}
+                - name: WIZ_ENV
+                  value: {{ coalesce .Values.global.wizApiToken.clientEndpoint .Values.wizApiToken.clientEndpoint | quote }}
+              {{- if or .Values.global.httpProxyConfiguration.enabled .Values.httpProxyConfiguration.enabled }}
+                - name: HTTP_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpProxy
+                      optional: false
+                - name: HTTPS_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpsProxy
+                      optional: false
+                - name: NO_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: noProxyAddress
+                      optional: false
+              {{- end }}
+              {{- if .Values.logLevel }}
+                - name: LOG_LEVEL
+                  value: {{ .Values.logLevel }}
+              {{- end }}
+              {{- with .Values.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with .Values.global.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- if .Values.podCustomEnvironmentVariablesFile }}
+                - name: CLI_ENV_FILE
+                  value: {{ .Values.podCustomEnvironmentVariablesFile }}
+                - name: USE_CLI_ENV_FILE
+                  value: "true"
+              {{- end }}
+                - name: WIZ_RUNTIME_METADATA_POD_NAME
```

Comment:
I couldn't find `WIZ_RUNTIME_METADATA_POD_NAME` `WIZ_RUNTIME_METADATA_NODE_NAME` `K8S_NAMESPACE` `WIZ_TERMINATION_GRACE_PERIOD` in the code. Where are they used in the manager? Can we remove them?